gclient/tests/memory_dump.rs

```diff
@@ -89,32 +89,27 @@ async fn memory_dump() -> Result<()> {
         .await?;
     assert!(listener.message_processed(message_id).await?.succeed());
 
-    assert_eq!(
-        charge_10(&api, program_id, &mut listener).await.unwrap(),
-        ""
-    );
+    assert_eq!(charge_10(&api, program_id, &mut listener).await?, "");
```

Comment:
Changing `unwrap()` -> `?` you removes information about a line where an error occurs.

Reply:
Agreed, those changes rolled back